modules/svg/rapid_features.js

```diff
@@ -356,6 +379,46 @@ export function svgRapidFeatures(projection, context, dispatch) {
   }
 
 
+  function drawViewfieldPoints(selection, viewfieldPoints) {
+    let viewfield = selection.selectAll("g.suggestionViewfieldgroup")
```

Comment:
let's stick to camel case:
`let viewField = selection.selectAll("g.suggestionViewFieldIconGroup")`